### DISCUSSION TITLE
注册的时候使用github登录会404

### DISCUSSION THREAD

**Discussion by CNCoreSteb** [BurnHair/docs#210] [技术支持]
rt，
![QQ_1732019531590](https://github.com/user-attachments/assets/0d211a56-e606-4d8c-9fe9-dd2b063d30d1)
我在邀请新朋友的时候遇到的，我自己测试也是这样，但是我已经注册的号且绑定了github如果我已经在浏览器中登录了github再选择github登录是没问题的

**Comment by BennyThink** [comment]
😨你的GitHub账号没有被 shadow ban吧？这个404的URL有吗？
